target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_bt_homehub-v5a.dts

```diff
@@ -262,12 +294,20 @@
 	status = "okay";
 	reset-gpios = <&gpio 21 GPIO_ACTIVE_LOW>;
 
-	wifi@168c,002d {
+	wifi@1 {
 		compatible = "pci168c,002d";
-		reg = <0x7000 0 0 0 0>;
-		qca,no-eeprom; /* load from ath9k-eeprom-pci-0000:00:0e.0.bin */
+		reg = <0x1000 0 0 0 0>;
+		nvmem-cells = <&cal_caldata_1000>, <&macaddr_caldata_110c 2>;
+		nvmem-cell-names = "calibration", "mac-address";
 		ieee80211-freq-limit = <2402000 2482000>;
 	};
+
+	wifi@2 {
+		compatible = "qcom,ath10k";
+		reg = <0x2000 0 0 0 0>;
+		nvmem-cells = <&cal_caldata_5000>, <&macaddr_caldata_110c 3>;
+		nvmem-cell-names = "calibration", "mac-address";
+	};
```

Comment:
this is not correct, the ath10k card is on the PCIe bus (`&pcie0`)

please note that the PCIe driver unfortunately is an out-of-tree one and I don't recall if it handles device-tree child nodes (at all - or properly).
so we need someone to test the result of this series (there was a big Home Hub 5A community - not sure if that's still the case)

Reply:
Oh that is very unfortunate.

Reply:
ping @alexceltare2 @sch-m @jaimet